coreos-base/oem-gce/files/units/oem-gce-enable-oslogin.service

```diff
@@ -0,0 +1,11 @@
+[Unit]
+DefaultDependencies=false
+Requires=sysinit.target
+After=sysinit.target
```

Comment:
Does this need to be ordered before sshd can start?

Reply:
Doesn't appear so. When I was first testing it I added the files manually and sshd seemed to be fine picking up the changes without restart. I still wanted it pretty early in the boot process just in case.

Reply:
Since we're socket-activating sshd by default, there is still the possibility of a user making an SSH connection before this script has run.